src/plugins/intel_gpu/src/plugin/ops/split.cpp

```diff
@@ -27,6 +27,13 @@ static void CreateCommonSplitOp(ProgramBuilder& p, const std::shared_ptr<ov::Nod
             ov::Shape start_offset(input_pshape.size());
             for (size_t i = 0; i < op->get_output_size(); i++) {
                 const auto outPartialShape = op->get_output_partial_shape(i);
+                // op->is_dynamic() does not check if output shape is dynamic. it only check dynamism for input shapes
+                // Even if op->is_dynamic() is false, output shape can be dynamic.
+                // Thus, it is necessary to check if output shape is dynamic.
+                if (outPartialShape.is_dynamic()) {
+                    offsets.clear();
+                    break;
```

Comment:
Shoundn't we prevent to go into the loop (at line28) at all and make it to proceed to the dynamic pass?

Reply:
Fixed it. thanks.